### PR TITLE
 [14.0] Migrate module web_m2x_options to v14

### PR DESCRIPTION
**Pull request by dsolanki-initos**
[14.0] Migrate module web m2x options to v14

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0 for all the pre-commit stuff.

**Comment by pedrobaeza** [comment]
I'm afraid the order of the pre-commit is not correct: it should be first before the v14 migration

**Comment by dsolanki-initos** [comment]
> I'm afraid the order of the pre-commit is not correct: it should be first before the v14 migration

I will change the sequence of the commit

**Review by ortlam — APPROVED** [review]

**Comment by dsolanki-initos** [comment]
@all
Can anyone please review the PR?

**Comment by ioans73** [comment]
Functional review

**Review by ioans73 — APPROVED** [review]

**Comment by pedrobaeza** [comment]
There's a pre-commit problem:
```
 /home/runner/work/web/web/web_m2x_options/static/src/js/form.js
  176:25  warning  Arrow function has a complexity of 38. Maximum allowed is 15  complexity
```
Can you please rebase and fix (or give instructions to ignore) such problem?

**Comment by fmdl** [comment]
@pedrobaeza it is possible to merge ?

File: web_m2x_options/__manifest__.py [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-    "version": "13.0.1.0.2",
+    "version": "14.0.1.0.0",
     "category": "Web",
-    "author": "ACSONE SA/NV, "
+    "author": "initOS GmbH,"
+    "ACSONE SA/NV, "
     "0k.io, "
     "Tecnativa, "
-    "Odoo Community Association (OCA)",
+    "Odoo Community Association (OCA),",
```
Comment (pedrobaeza) [inline web_m2x_options/__manifest__.py]: This final comma is incorrect

**Pull request merged**